docs/GettingStarted.md

```diff
@@ -2466,3 +2469,29 @@ However, additional instrumentation provided by Datadog can be activated alongsi
 | `OpenTracing::FORMAT_TEXT_MAP` | Yes        |                        |
 | `OpenTracing::FORMAT_RACK`     | Yes        | Because of the loss of resolution in the Rack format, please note that baggage items with names containing either upper case characters or `-` will be converted to lower case and `_` in a round-trip respectively. We recommend avoiding these characters or accommodating accordingly on the receiving end. |
 | `OpenTracing::FORMAT_BINARY`   | No         |                        |
+
+## Known issues and suggested configurations
+
+### Payload too large
+
+By default, Datadog limits the size of trace payloads to prevent memory overhead within instrumented applications. However, if your application creates long-running traces, measured in minutes, or traces with thousands of operations, these traces may not be sent to Datadog due to these size limits.
```

Comment:
```suggestion
By default, Datadog limits the size of trace payloads to prevent memory overhead within instrumented applications. However, if your application creates long-running traces, or traces with thousands of operations, these traces may not be sent to Datadog due to these size limits.
```